### PR TITLE
Update dist files

### PR DESCRIPTION
**Pull request by Mcdavid95**
Peer review for Technical-lead

### CODE REVIEW

**Review by ayodelevm — COMMENTED** [review]
Good work so far. I think the use of "malformed file" may not be descriptive enough of the error message you are trying to pass across, since it is testing to know if the JSON Array supplied is in accordance with a particular format. something like "Incorrect JSON Array format" may communicate the actual error better to the user.

File: .env.example [hunk 1]
```diff
@@ -1,4 +1,5 @@
 NODE_ENV = DEV
 NODE_ENV = TEST
 PORT_DEV = 3000
```
Comment (codingpop) [inline .env.example:3]: This shouldn't be in your .env.example

File: .env.example [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
-PORT_TEST = 8080
\ No newline at end of file
+PORT_TEST = 8080
```
Comment (codingpop) [inline .env.example:4]: Even this shouldn't be here

File: .env.example [hunk 3]
```diff
@@ -1,4 +1,5 @@
...
+CODACY_PROJECT_TOKEN = '192bff7bac8b4925a07779f3eafc7f8c'
```
Comment (codingpop) [inline .env.example:5]: What does this do? Is it really necessary?

**Pull request merged**